hw/cortexm/graphic.c

```diff
@@ -718,3 +720,22 @@ static void cortexm_graphic_led_turn(BoardGraphicContext *board_graphic_context,
 
 // ----------------------------------------------------------------------------
 
+#else
```

Comment:
A more consistent approach would be to not define these empty functions and remove the references to them. Would this be complicated?

Reply:
There were like 10 references to them. I thought declaring them as empty would be easier. Can provide another variant of the patch and see what is cleaner.